packages/atomic/src/pages/header.js

```diff
@@ -0,0 +1,36 @@
+const links = [
+  {href: '/', label: 'Main'},
+  {href: '/examples/custom.html', label: 'Custom'},
+  {href: '/examples/external.html', label: 'External'},
+  {href: '/examples/folding.html', label: 'Folding'},
+  {href: '/examples/headless.html', label: 'Headless'},
+  {href: '/examples/standalone.html', label: 'Standalone'},
+];
+
+const header = document.createElement('header');
+
+const getCurrentExample = () =>
+  links.find((link) => link.href === window.location.pathname);
+
+const makeLinks = () => {
+  return links
+    .filter((link) => link.href !== window.location.pathname)
+    .map(
+      (link) =>
+        `<a href="${link.href}" style="margin-right: 10px; color: var(--atomic-primary); text-decoration: none;">${link.label}</a>`
```

Comment:
I would wrap the `a` element in a `li` (even though accessibility doesn't matter that much here)

Reply:
ah yeah, that's what I meant to do! thanks for catching it